api-gateway/src/main/resources/application-dev.yml

```diff
@@ -7,7 +7,7 @@ gateway:
   sandbox:
     url: http://localhost:5400
   demo:
-    url: http://localhost:5401
+    url: http://localhost:5421
```

Comment:
can it be just 5420?
because we have additional port: `managementPort` which is `port` + 1 usually

Reply:
I would not like to allow ports that ends with `0` as far as they seem to be more popular.
`managementPort` will be `5422`, why not? Is it occupied?

Reply:
common pattern for all application: port ends on `0`, `managementPort` ends on `1`.
Now we have exception for demo project, which is not a temporary application in our ecosystem

Reply:
Suggestion: use 5-10 ports per service (one is container port, one is management port and others are reserved) going one by one: let's say from `12120` or some other port that is not that nice as `5000`

Reply:
looks ok for me
@petertrr @akuleshov7

Reply:
I think it doesn't matter that much, especially in Kubernetes, where ports just need to match in `application.properties` and `Deployment`, and then the port can be referenced by name and Service provides access on port 80.
We can even keep existing ports for development and then just override `server.port` in ConfigMap with `{{ .service.containerPort }}` to make change smaller

Reply:
> We can even keep existing ports for development and then just override `server.port` in ConfigMap with `{{ .service.containerPort }}` to make change smaller

we can, but need to add ports for dev profile to be able run locally

Reply:
Anyway it seems to be a good practice for local deploy. It is better to reserve `n` ports going one by one then to reserve randomly-chosen ports